styles/style.css

```diff
@@ -296,6 +296,76 @@ ul {
 
 /*Main end*/
 
+/*Section Homes start*/
+.section_homes {
+    background-color: #EAF0F9;
+    text-align: center;
+}
+
+.container_homes {
+    max-width: 1232px;
+    padding: 120px 0;
+    margin: 0 auto;
+}
+
+.homes_title_wrap {
+    margin: 0 auto;
+    padding-bottom: 104px;
+}
+
+.homes_title {
+    font-style: normal;
+    font-weight: 500;
+    font-size: 40px;
+    line-height: 47px;
+    color: #383838;
+}
+
+.homes_picture {
```

Comment:
you can use flex-wrap: nowrap, and then add to your picture card margin-right:16px except last, than your cards will be more accurtate

Reply:
You forgot about gap between cards